code/API_definitions/quality-on-demand.yaml

```diff
@@ -188,6 +190,8 @@ paths:
                   $ref: "#/components/responses/Generic403"
                 "410":
                   $ref: "#/components/responses/Generic410"
+                "422":
```

Comment:
This is the notification callback, you have to add the new enum to the already existent 422 response at line 214 (you would have to substitute the Generic422)